tests/fsharpqa/Source/Conformance/SpecialAttributesAndTypes/Imported/CallerInfo/CallerMemberName.fs

```diff
@@ -45,6 +47,8 @@ type MyTy() =
             |> List.head
         MyTy.Check(result, Some("CheckMembers"), "lambda")
         MyTy.Check(functionVal (), Some("functionVal"), "functionVal")
+        MyTy.Check(anonymousLambda, Some("anonymousLambda"), "anonymousLambda")
+        MyTy.Check(asyncVal, Some("asyncVal"), "asyncVal")
```

Comment:
`do` in a primary ctor is covered (returns ".ctor") https://github.com/Microsoft/visualfsharp/pull/1114/files#diff-7e9f3ca9172a26b1556a899077a98bfcR22

Not with explicit `do` keyword, but top-level effect in module is covered here: https://github.com/Microsoft/visualfsharp/pull/1114/files#diff-7e9f3ca9172a26b1556a899077a98bfcR111  (returns ".cctor" which is defensible, though not ideal. Returning the module name would be better IMO)

Explicit `static do` should be added.  All of this definitely needs to be added to RFC.

Static